src/index.ts

```diff
@@ -59,6 +62,8 @@ export * as BOBasePage from '@pages/BO/BOBasePage';
 export {default as boLoginPage} from '@pages/BO/login';
 export {default as boDashboardPage} from '@pages/BO/dashboard';
 export {default as boOrdersPage} from '@pages/BO/orders';
+export {default as boOrdersBasePage} from '@pages/BO/orders/view/viewOrderBasePage';
```

Comment:
```suggestion
export {default as boOrdersViewBasePage} from '@pages/BO/orders/view/viewOrderBasePage';
```

I recommend it as we use it as a base for boOrdersView pages.